quadriga/src/main/java/edu/asu/diging/quadriga/core/model/events/pattern/PatternAppellationEvent.java

```diff
@@ -0,0 +1,24 @@
+package edu.asu.diging.quadriga.core.model.events.pattern;
```

Comment:
these are just temporary classes created from the submitted pattern file, right? They should not be in the model package.